_includes/tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/create-topic.sh

```diff
@@ -0,0 +1 @@
+docker exec -i broker /usr/bin/kafka-topics --create --topic example-topic --zookeeper zookeeper:2181 --replication-factor 1 --partitions 1
```

Comment:
We should use the `bootstrap-server` arg instead of `zookeeper`

Reply:
good catch, fixed